lang/en/qtype_questionpy.php

```diff
@@ -34,6 +34,9 @@
 $string['max_package_size_kb'] = 'Maximum file size of a QuestionPy package';
 $string['max_package_size_kb_description'] = 'Maximum file size in kB';
 $string['open_website'] = 'Open website';
+$string['options_form_validation_error_container'] = 'The following errors occurred while validating the form and could not'
+    . ' be mapped to an input element: <ul>{$a}</ul>';
+$string['options_form_validation_error_element'] = '<li>{$a->name}: {$a->error}</li>';
```

Comment:
Dazu steht zwar nichts in Moodle's Coding Style, ich persönlich würde so strukturelle HTML-Elemente (also ich meine jetzt nicht `<b>` und `<a>`) lieber außerhalb der Lang-Strings sehen, wenn es keinen Mehraufwand bedeutet (und ich denke das würde es hier nicht.)